src/MacroExpander.js

```diff
@@ -156,9 +142,7 @@ class MacroExpander {
                         // expansion.splice(i, 2, arg[0], arg[1], …)
                         // to replace placeholder with the indicated argument.
                         // TODO: use spread once we move to ES2015
-                        expansion.splice.apply(
-                            expansion,
-                            [i, 2].concat(args[tok.text - 1]));
+                        expansion.splice(i, 2, ...args[tok.text - 1]);
```

Comment:
Can run `make build` and check what's being output for the spread operator.  Also, if we go with this, the `TODO` will no longer be necessary.

Reply:
Ooops! I meant to remove it. I'd failed to do this part when manually merging in my changes with Erik's previous commit. Indeed the results produced by `make` seem sane:
```javascript
var _expansion;
// replace placeholder with the indicated argument.
(_expansion = expansion).splice.apply(_expansion, [_i, 2].concat((0, _toConsumableArray3.default)(args[_tok.text - 1])));
```